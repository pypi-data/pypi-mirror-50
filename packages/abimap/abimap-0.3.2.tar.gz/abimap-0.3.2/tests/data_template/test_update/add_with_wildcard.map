# This map file was updated with PROGRAM_NAME_VERSION

WITH_WILDCARD_1_0_0
{
    global:
        one_symbol;
        *;
    local:
        *;
} ;


