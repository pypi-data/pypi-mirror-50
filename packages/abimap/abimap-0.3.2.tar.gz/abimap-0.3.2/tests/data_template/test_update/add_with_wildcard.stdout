Added:
    symbol

# This map file was updated with PROGRAM_NAME_VERSION

WITH_WILDCARD_1_0_0
{
    global:
        *;
        one_symbol;
    local:
        *;
} ;

WITH_WILDCARD_1_1_0
{
    global:
        symbol;
} WITH_WILDCARD_1_0_0;

